apps/rush-lib/src/logic/buildCache/ProjectBuildCache.ts

```diff
@@ -272,10 +265,11 @@ export class ProjectBuildCache {
       const logFilePath: string = this._getTarLogFilePath();
       const tarExitCode: number = await tarUtility.tryCreateArchiveFromProjectPathsAsync({
         archivePath: tempLocalCacheEntryPath,
-        paths: filesToCache.outputFilePaths,
+        paths: foldersToCache,
```

Comment:
The `paths` property should probably get renamed to make it clear that it's a list of folder names.

Reply:
It can be folders or files, the Tar exe doesn't care. It will walk folders, or write files.